### PR TITLE
#TP-13 ft(register): driver and operator

### PR DESCRIPTION
**Pull request by chanceDira**
## What this PR Do?

This PR is for showing our progress on the tasks below

- [x] The administrator should be able to register a new driver or a new operator.
- [x] The **driver** and **operator** should receive email with password.
- [x] Appropriate error messages are displayed on unsuccessful attempts.

WIP #TP-13

## Testing Repo

**Clone the repo**
then navigate to **ft-register-drivers-operator-TP-13** branch

**Installation of dependencies.**
```
npm i
```
**Run development**
```
npm run dev
```

## Trello
https://trello.com/c/H8QA9wLO/13-register-drivers-operators

### CODE REVIEW

File: src/containers/RegisterDriverOperator/RegisterOperator.js [hunk 1]
```diff
@@ -18,13 +18,15 @@ import close from '../../assets/svgs/close.svg';
 import prev from '../../assets/svgs/prev.svg';
 import next from '../../assets/svgs/next.svg';
 import { useSelector } from 'react-redux';
+import { API as axios } from '../../api/index.js';
```
Comment (sezeranoJchrisostome) [inline src/containers/RegisterDriverOperator/RegisterOperator.js:21]: I like this approach of having the same configuration

File: src/api/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios"
+
+const API = axios.create({ 
+    baseURL: "http://localhost:5000/api/v1",
```
Comment (sezeranoJchrisostome) [inline src/api/index.js:4]: can we have this in the env file so that we may modify this link later on in production

**Review by sezeranoJchrisostome — CHANGES_REQUESTED** [review]

**Review by sezeranoJchrisostome — CHANGES_REQUESTED** [review]
Well done @chanceDira  this was good work!

File: src/api/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (chanceDira) [reply src/api/index.js]: _Thank you @sezeranoJchrisostome. I have applied you feedback and it is working fine._

**Pull request closed without merging**